packages/elements/src/photon-authorized/index.tsx

```diff
@@ -0,0 +1,37 @@
+//Shoelace
+import '@shoelace-style/shoelace/dist/components/dialog/dialog';
+import { setBasePath } from '@shoelace-style/shoelace/dist/utilities/base-path.js';
+
+setBasePath('https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.0.0-beta.82/dist/');
+
+import { createEffect, createSignal, JSXElement, Show } from 'solid-js';
+import { usePhoton } from '../context';
+
+export const PhotonAuthorized = (props: { children: JSXElement }) => {
+  const client = usePhoton();
+  const [authorized, setAuthorized] = createSignal<boolean>(
+    client?.authentication.state.isAuthorized || false
+  );
+  const [isLoading, setIsLoading] = createSignal<boolean>(
+    client?.authentication.state.isLoading || false
+  );
+  createEffect(() => {
+    setAuthorized(client?.authentication.state.isAuthorized || false);
+  });
+  createEffect(() => {
+    setIsLoading(client?.authentication.state.isLoading || false);
+  });
+
+  return (
+    <>
+      <Show when={client && !authorized() && !isLoading()}>
+        <sl-alert variant="warning" open>
+          <sl-icon slot="icon" name="exclamation-triangle"></sl-icon>
+          <strong>You are not authorized to prescribe</strong>
```

Comment:
@samkotlove  update this to say something like please contact support at Photon?

Reply:
Yeah I would keep it broad. "something went wrong" or w/e